CONTRIBUTING.md

```diff
@@ -45,11 +44,9 @@ As above, a critical component of maintaining support and ensuring the wide numb
 
 ### Compile MCprep using scripts
 
-Scripts have been created for Mac OSX (`compile.sh`) and Windows (`compile.bat`) which make it fast to copy the entire addon structure the addon folders for multiple versions of blender. You need to use these scripts, or at the very least validate that they work, as running the automated tests depend on them.
+MCprep uses the [bpy-addon-build](https://github.com/StandingPadAnimations/bpy-build) package to build the addon, which makes it fast to copy the entire addon structure the addon folders for multiple versions of blender.
```

Comment:
Also, is `mcprep_venv_*` meant to be your virtual environment name? I suppose we should use that above instead of `venv` since we already have this in the gitignore (would be nicer if we could update the gitignore to be `mcprep_venv*` so it could also match `mcprep_venv` itself)